src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/transformation/FormAToCaseTransformer.java

```diff
@@ -54,9 +56,33 @@
         applyMappingsForAddress("respondent", ocrDataFields, transformedCaseData);
         AddressMapper.applyMappings("respondentSolicitor", "rSolicitorAddress", ocrDataFields, transformedCaseData);
 
+        mapAuthorisationSignedToYesOrNo(OcrFieldName.AUTHORISATION_SIGNED, "authorisationSigned", ocrDataFields, transformedCaseData);
+        
+        mapFormDateToCcdDate(OcrFieldName.AUTHORISATION_DATE, "authorisation3", ocrDataFields, transformedCaseData);
+        
         return transformedCaseData;
     }
 
+    private void mapFormDateToCcdDate(String ocrFieldName, String ccdFieldName,
+                                      List<OcrDataField> ocrDataFields, Map<String, Object> formSpecificMap) {
+        getValueFromOcrDataFields(ocrFieldName, ocrDataFields).ifPresent(ocrAuthorisationDate -> {
+            String ccdAuthorisationDate = transformFormDateIntoCcdDate(OcrFieldName.AUTHORISATION_DATE, ocrAuthorisationDate);
+            formSpecificMap.put(ccdFieldName, ccdAuthorisationDate);
+        });
+    }
+    
+    private void mapAuthorisationSignedToYesOrNo(String ocrFieldName, String ccdFieldName,
+                                                 List<OcrDataField> ocrDataFields, Map<String, Object> formSpecificMap) {
+        ocrDataFields.stream()
+            .filter(ocrDataField -> ocrDataField.getName().equals(ocrFieldName))
+            .map(OcrDataField::getValue)
+            .findFirst()
+            .ifPresent(ocrValue -> {
+                String ccdValue = ocrValue.trim().isEmpty() ? "No" : "Yes";
```

Comment:
We have constants for this (at least in Divorce).

Reply:
```
uk.gov.hmcts.reform.finrem.caseorchestration.OrchestrationConstants.YES_VALUE
uk.gov.hmcts.reform.finrem.caseorchestration.OrchestrationConstants.NO_VALUE
```